### PR TITLE
Updating OME-TIFF & XML docs for 2016-06

### PR DESCRIPTION
**Pull request by hflynn**
This attempts to update all the sample OME-XML from the Model & Formats docs. The sample files info is copied from https://github.com/sbesson/samples/tree/2016-06_samples but I've guessed the compliant file changes so please check them properly (i.e the whole file not just my changes)! Also, the samples on https://www.openmicroscopy.org/site/support/ome-model-staging/developers/using-ome-xml.html will need more work because I notice one is about LightSource (see https://www.openmicroscopy.org/site/support/ome-model-staging/developers/using-ome-xml.html#sample-third-party-pieces-xml) and also, again I've made an educated guess at how to update them.

### CODE REVIEW

**Comment by hflynn** [comment]
PRs to my branch welcome if that's easiest

File: formats/ome-tiff/specification.txt [hunk 1]
```diff
@@ -340,64 +340,66 @@ distributed across multiple files. Each of the files in the set has identical
...
-         xsi:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2015-01
-                             http://www.openmicroscopy.org/Schemas/OME/2015-01/ome.xsd">
+         Creator="OME Bio-Formats 5.2.0-m4"
+         UUID="urn:uuid:45c8bf18-6aa2-478c-9080-e0b0d3eb1f70"
+         xsi:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2016-06
+         http://www.openmicroscopy.org/Schemas/OME/2016-06/ome.xsd">
     ...
-        <Pixels BigEndian="false" DimensionOrder="XYZTC" ID="Pixels:0"
-                Interleaved="false" SignificantBits="8" SizeC="2" SizeT="43"
+        <Pixels BigEndian="false" DimensionOrder="XYZTC" ID="Pixels:0"
```
Comment (ghost) [inline formats/ome-tiff/specification.txt]: trailing whitespace

File: formats/ome-tiff/specification.txt [hunk 2]
```diff
@@ -411,18 +413,14 @@ OME-TIFF files using the
...
+    <?xml version="1.0" encoding="UTF-8"?>
+    <OME UUID="urn:uuid:4978087c-a670-4b12-af53-256c62d8d101"
+         xmlns="http://www.openmicroscopy.org/Schemas/OME/2016-06"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2015-01
-                             http://www.openmicroscopy.org/Schemas/OME/2015-01/ome.xsd">
-       <BinaryOnly MetadataFile="multifile-Z1.ome.tiff"
-                   UUID="urn:uuid:4978087c-a670-4b12-af53-256c62d8d101"/>
+         xsi:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2016-06
+            http://www.openmicroscopy.org/Schemas/OME/2016-06/ome.xsd">
```
Comment (ghost) [inline formats/ome-tiff/specification.txt]: loss of alignment with prior line

File: formats/ome-tiff/specification.txt [hunk 3]
```diff
@@ -340,64 +340,66 @@ distributed across multiple files. Each of the files in the set has identical
...
-              urn:uuid:69bfd94f-2d95-41ae-887d-26e0c11fd606
+              urn:uuid:45c8bf18-6aa2-478c-9080-e0b0d3eb1f70
             </UUID>
           </TiffData>
     ...
-          <TiffData FirstC="0" FirstT="1" FirstZ="0" IFD="0" PlaneCount="1">
-            <UUID FileName="tubhiswt_C0_TP1.ome.tif">
-              urn:uuid:16093e77-8531-43c5-83df-c71efbd212d0
+          <TiffData FirstC="0" FirstT="0" FirstZ="1" IFD="1" PlaneCount="1">
+            <UUID FileName="tubhiswt_C0_TP0.ome.tif">
```
Comment (dgault) [inline formats/ome-tiff/specification.txt]: Should this be UUID: FileName="tubhiswt_C0_TP1.ome.tif"

Comment (sbesson) [inline formats/ome-tiff/specification.txt]: I think so. The goal of this snippet was to show different `TiffData` element pointing to different files on disk - might need to update the `FirstT/FirstZ` as well.

Comment (hflynn) [inline formats/ome-tiff/specification.txt]: Ah! I was just saying to @dgault in slack that it looked right when I doublechecked but yes, I could change the FirstT value too

**Comment by hflynn** [comment]
Right, should be sorted now @sbesson I will rebuild staging once Travis is green

**Comment by sbesson** [comment]
In `formats/specifications/minimum.txt`, we should drop the namespaces now the schemas are unified i.e. `<ome:TiffData>` -> `<TiffData>` etc

**Comment by hflynn** [comment]
Should have caught all of those now!

**Comment by sbesson** [comment]
All looks good on the staging pages now.

**Pull request merged**